content/source/docs/extend/sdkv2-intro.html.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: "extend"
+page_title: "Home - Plugin Development: SDKv2"
+description: |-
+  Learn about version 2 of the Terraform Plugin SDK.
+---
+
+# Terraform Plugin SDKv2
+
+Terraform Plugin SDKv2 is an established way to develop Terraform Plugins. It is the precursor to the [Terraform Plugin Framework](/docs/plugin/framework/index.html), which is still in early development.
```

Comment:
@paddycarver Noted! I updated this a bit to try to get closer to what you described - the SDKv2 is still established and in many cases preferred, but that the framework is coming! and then I opted away from making a specific recommendation and just point folks toward the which SDK page to learn more.

Reply:
As long as @aareet or @redeux is on board with this framing, it works for me.

Reply:
Thank you! @aareet I tagged you for review - let me know what you think when you get a chance!

Reply:
Looks good to me, thank you for writing all this up!